sample-configurations/foak-fast-kernels-sample-configuration.yaml

```diff
@@ -22,10 +22,13 @@ plugins:
       # - the FastQuantized version is all-or-nothing
 
       # fast loss triton kernels
-      fast_loss: True
+      fast_loss: False
 
       # fast rms norm triton kernels
       fast_rsm_layernorm: True
 
       # fast RoPE embedding triton kernels
       fast_rope_embeddings: True
+
+      # fused linear cross entropy loss
+      fused_linear_loss: True
```

Comment:
I think this is problematic because `fast_loss` and `fused_linear_loss` both switch the loss, so it causes weird combinations

Maybe its better to convert `fast_loss` to accept both boolean and a string

`fast_loss: True` means legacy `fast_loss: 'fused'` means fused

Reply:
or use `fast_loss` = True / False and `fast_loss_type` we can have the deafult (non-fused) and `"fused"`

Reply:
But that's why we are only allowing the user to set one, if one is true, the other must be false. That prevents this issue right?

Reply:
I just feel that if already having `True/False` switches is already abit annoying as you mentioned before. Now having constraints between them might be even more confusing